source/module_base/math_sphbes.h

```diff
@@ -14,26 +14,53 @@ class Sphbes
     Sphbes();
     ~Sphbes();
 
+    /**
+     * @brief spherical bessel
+     * 
+     * @param msh [in] number of grid points
+     * @param r [in] radial grid (1:msh)
+     * @param q [in] k_radial
+     * @param l [in] angular momentum
+     * @param jl [out] jl(1:msh) spherical bessel function
+     */
     static void Spherical_Bessel
     (
-        const int &msh,	//number of grid points
-        const double *r,//radial grid
-        const double &q,	//
-        const int &l,	//angular momentum
-        double *jl	//jl(1:msh) = j_l(q*r(i)),spherical bessel function
+        const int &msh,
+        const double *r,
+        const double &q,
+        const int &l,
+        double *jl
     );
 
+    /**
+     * @brief spherical bessel
+     * 
+     * @param msh [in] number of grid points
+     * @param r [in] radial grid (1:msh)
```

Comment:
Sorry but I can't get the meaning of `(1:msh)` here. Could you explain it to me?

Reply:
the index is 1 to msh, may be this is redundant. I think it is better to delete it.